web/src/views/live/LiveDashboardView.tsx

```diff
@@ -247,6 +252,34 @@ export default function LiveDashboardView({
     [setPreferredLiveModes],
   );
 
+  const onKeyboardShortcut = useCallback(
+    (key: string, modifiers: KeyModifiers) => {
+      if (!modifiers.down) {
+        return;
+      }
+
+      switch (key) {
+        case "f":
+          toggleFullscreen();
+          break;
+      }
+    },
+    [toggleFullscreen],
+  );
+
+  useKeyboardListener(["f"], onKeyboardShortcut);
+  const handleFullScreenQuery = useCallback(
+    (value: string) => {
+      const desiredState = value === "true";
+      if (fullscreen === desiredState) {
+        return;
+      }
+      toggleFullscreen;
```

Comment:
```suggestion
      if (!fullscreen) {
          toggleFullscreen();
      }
```

Reply:
not showing in full screen in the default state so I think it is safe to assume that if the query arg is provided the user wants fullscreen

also, toggleFullscreen needs to be called not just passed.

Reply:
Should we check that the value `=== "true"`? Assuming someone can programmatically open the URL with it being either true or false?
so in case of fullscreen=false we just ignore it